src/test/java/jp/ac/utokyo/rcast/karkinos/wavelet/WaveletDenoizeTest.java

```diff
@@ -0,0 +1,669 @@
+package jp.ac.utokyo.rcast.karkinos.wavelet;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.apache.commons.math.stat.descriptive.SummaryStatistics;
+import manifold.ext.rt.api.Jailbreak;
+
+import jp.ac.utokyo.rcast.karkinos.exec.CapInterval;
+import jp.ac.utokyo.rcast.karkinos.exec.CNVInfo;
+import jp.ac.utokyo.rcast.karkinos.exec.DataSet;
+
+public class WaveletDenoizeTest {
+    static Stream<Arguments> calcParameters() {
+        Map<String, Integer> dummyIndex = new HashMap<>();
+        @Jailbreak DataSet dataset1 = new DataSet(dummyIndex);
+        @Jailbreak DataSet dataset2 = new DataSet(dummyIndex);
```

Comment:
We don't need this map, right?
```suggestion
        @Jailbreak DataSet dataset1 = new DataSet(null);
        @Jailbreak DataSet dataset2 = new DataSet(null);
```